src/app.js

```diff
@@ -28,16 +28,16 @@ const structuredResults = connectHits(({ hits, widgetParams }) => {
     result._rankingInfo.words - 1 === result._rankingInfo.proximityDistance
   ) {
     container.innerHTML = `
-      <aside>
-        <img src="${result.url}" alt=${result.name}>
-        <h3>${result.name}</h3>
-      <aside>
+      <img class="image" src="${result.url}" alt=${result.name} />
+      <h3>${result.name}</h3>
     `;
     return;
   }
 
-  container.innerHTML =
-    '<p>No structured results for this query – try e.g. "Acer".</p>';
+  container.innerHTML = `
+    <div class="image"></div>
```

Comment:
I think it's OK to remove this. It's weirder than having a different height IMO.